boards/same54-xpro/include/periph_conf.h

```diff
@@ -106,6 +106,13 @@ static const tc32_conf_t timer_config[] = {
 #define TIMER_NUMOF         ARRAY_SIZE(timer_config)
 /** @} */
 
+/**
+ * @name ATA6561 STANDBY pin definition
```

Comment:
```suggestion
 * @brief ATA6561 STANDBY pin definition
```
maybe this makes doxygen happy